.scripts/nightly.yml

```diff
@@ -0,0 +1,29 @@
+schedules:
+  - cron: "0 0 * * *"
+    displayName: Daily midnight build
+    branches:
+      include:
+        - main
+
+steps:
+  - script: npm install -g @typespec/compiler@next
+    displayName: Install @typespec/compiler@next
+  - template: build.yml
+  - script: npm run override-dev-version
+    workingDirectory: $(Build.SourcesDirectory)/packages/typespec-ts
+    displayName: "Override pnpm-config version to latest dev"
+  - script: npx @azure-tools/typespec-bump-deps package.json
```

Comment:
Not very clear why we still need this if we have override-dev-version script ?

Reply:
If we don't add this the command `npm list` would fail